doc/whats_new.rst

```diff
@@ -91,6 +91,10 @@ API
 
     - :class:`mne.EpochsArray` no longer has an average EEG reference silently added (but not applied to the data) by default. Use :func:`mne.EpochsArray.add_eeg_average_proj` to properly add one.
 
+    - Deprecated functions :func:`mne.time_frequency.compute_raw_psd` and :func:`mne.time_frequency_compute_epochs_psd`, replaced by :func:`mne.time_frequency.psd_multitaper` and :func:`mne.time_frequency.psd_welch` by `Chris Holdgraf`_
+
+    - Deprecated function :func:`mne.time_frequency.multitaper_psd` and replaced by :func:`mne.time_frequency._psd_multitaper` by `Chris Holdgraf`_
+
```

Comment:
Hmmmm, there was some weirdness with how picking worked but I'm not sure specifically what the bug was (if it was a bug)...not sure what else it would be